### ISSUE TITLE
Numerous error sounds are played when running `npm install` in Windows Terminal

### ISSUE THREAD

**Issue by Neurrone** [nvaccess/nvda#12723]
/CC @codeofdusk 

### Steps to reproduce:

1. Run `npm i` for an NPM project in windows terminal

### Actual behavior:

No error sounds are heard

### Expected behavior:

Error sounds are heard continuously with the following stack trace for as long as the npm install command is running:

```
ERROR - eventHandler.executeEvent (10:19:26.854) - MainThread (11520):
error executing event: caret on <NVDAObjects.Dynamic_WinTerminalUIAEditableTextWithAutoSelectDetectionUIA object at 0x00C72C70> with extra args of {}
Traceback (most recent call last):
  File "eventHandler.pyc", line 284, in executeEvent
  File "eventHandler.pyc", line 98, in __init__
  File "eventHandler.pyc", line 107, in next
  File "NVDAObjects\behaviors.pyc", line 195, in event_caret
  File "NVDAObjects\__init__.pyc", line 1213, in event_caret
  File "review.pyc", line 174, in handleCaretMove
  File "documentBase.pyc", line 22, in makeTextInfo
  File "NVDAObjects\UIA\__init__.pyc", line 312, in __init__
  File "NVDAObjects\UIA\__init__.pyc", line 792, in collapse
  File "monkeyPatches\comtypesMonkeyPatches.pyc", line 27, in __call__
_ctypes.COMError: (-2147467259, 'Unspecified error', (None, None, None, 0, None))
ERROR - eventHandler.executeEvent (10:19:26.935) - MainThread (11520):
error executing event: caret on <NVDAObjects.Dynamic_WinTerminalUIAEditableTextWithAutoSelectDetectionUIA object at 0x00C72C70> with extra args of {}
Traceback (most recent call last):
  File "eventHandler.pyc", line 284, in executeEvent
  File "eventHandler.pyc", line 98, in __init__
  File "eventHandler.pyc", line 107, in next
  File "NVDAObjects\behaviors.pyc", line 195, in event_caret
  File "NVDAObjects\__init__.pyc", line 1213, in event_caret
  File "review.pyc", line 174, in handleCaretMove
  File "documentBase.pyc", line 22, in makeTextInfo
  File "NVDAObjects\UIA\__init__.pyc", line 312, in __init__
  File "NVDAObjects\UIA\__init__.pyc", line 792, in collapse
  File "monkeyPatches\comtypesMonkeyPatches.pyc", line 27, in __call__
_ctypes.COMError: (-2147467259, 'Unspecified error', (None, None, None, 0, None))
```

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
alpha-23553,3c29f9ad

#### Windows version:
Windows 10 20h2

#### Name and version of other software in use when reproducing the issue:
Windows terminal 1.9.1942.0
Npm 7.7.6, running in Ubuntu via WSL2

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No

#### If add-ons are disabled, is your problem still occurring?
Yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Not applicable

**Comment by codeofdusk** [comment]
CC @carlos-zamora.
It looks like NVDA is attempting to create degenerate UIA ranges and failing.

**Comment by Adriani90** [comment]
I cannot reproduce this with NVDA alpha-34581,32245d46 (2025.1.0.34581) on Windows 11 23 H2 and npm 10.9.1.
@Neurrone are you still having this issue with Windows terminal v1.21.2911.0 on Windows 10 or 11?

I guess this has been fixed by #14888.

**Comment by Neurrone** [comment]
I haven't seen this in a while and will close it.
